### PR TITLE
add more auto backup hours

### LINKED ISSUE #36: More option for backup frequency
### Describe your suggested feature

1h 3h...

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.10.5](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by kana-shii**
add 1h, 3h auto-backup options
closes #36

### CODE REVIEW

File: i18n/src/commonMain/moko-resources/base/strings.xml [hunk 1]
```diff
@@ -266,6 +266,8 @@
     <string name="pref_category_library_update">Global update</string>
     <string name="pref_library_update_interval">Automatic updates</string>
     <string name="update_never">Off</string>
+    <string name="update_1hour">Every 1 hour</string>
```
Comment (cuong-tran) [inline i18n/src/commonMain/moko-resources/base/strings.xml:269]: if you put the strings into MR, they won't get translated.
For them to be included in the Komikku's translation, please add them to KMR (i18n-kmk folder)

Comment (kana-shii) [reply i18n/src/commonMain/moko-resources/base/strings.xml:269]: ill change it rn

**Comment by cuong-tran** [comment]
Could you rebase and try again? I need to fix the PR build check for it to work.

**Comment by isabella-almeidaa** [comment]
> Could you rebase and try again? I need to fix the PR build check for it to work.

Sorry... Wdym by rebase? How do I do that exactly?

**Comment by kana-shii** [comment]
> Could you rebase and try again? I need to fix the PR build check for it to work.

should I rebase it to the default branch or create another one?

**Pull request merged**